src/components/CommandBar.js

```diff
@@ -62,10 +62,26 @@ function formatKey(action) {
   return formatKeyShortcut(key);
 }
 
-function debugBtn(onClick, type, className, tooltip) {
+function handlePressAnimation(button) {
+  if (button) {
+    button.style.opacity = "0";
+    button.style.transform = "scale(1.3)";
+    setTimeout(() => {
+      button.style.opacity = "1";
+      button.style.transform = "none";
+    }, 200);
+  }
+}
+
+function debugBtn(onClick, type, className, tooltip, getRefCallback) {
   className = `${type} ${className}`;
   return dom.span(
-    { onClick, className, key: type },
+    {
+      onClick,
+      className,
+      key: type,
+      ref: getRefCallback(type)
```

Comment:
you mean pass the type as a string right?
I can do it but the react team recommends the callback attribute approach

<img width="1083" alt="screen shot 2017-01-15 at 7 41 51 pm" src="https://cloud.githubusercontent.com/assets/6891569/21964749/e63a7a1e-db5a-11e6-9526-08e71aa3b0ad.png">